stdlib/2and3/argparse.pyi

```diff
@@ -114,21 +114,31 @@ class ArgumentDefaultsHelpFormatter(HelpFormatter): ...
 if sys.version_info >= (3,):
     class MetavarTypeHelpFormatter(HelpFormatter): ...
 
-class Action:
+class Action(Generic[_T]):
```

Comment:
Does this need to be generic? At least in this file, all the occurrences are just `Action`, not `Action[SomeType]`.

Reply:
I made the class generic because the 'type' and 'choices' attributes use a type parameter.  Would it be better to just replace the type parameter with Any?

Reply:
If we make it generic, we should also try to ensure that all usages pass the type parameter. If that's too hard, that's a sign that maybe it's better to just use Any.

Reply:
Good point. I tried changing the usages and found myself mostly using Action[Any], so I undid the generic change.